pennylane/decomposition/symbolic_decomposition.py

```diff
@@ -131,6 +131,49 @@ def adjoint_controlled_decomp(*_, base, **__):
     )
 
 
+def _adjoint_pow_resource(base_class, base_params):  # pylint: disable=unused-argument
+    """Resources of the adjoint of the power of a gate."""
+    base, base_params, z = base_params["base_class"], base_params["base_params"], base_params["z"]
+    return {pow_resource_rep(base, base_params, z): 1}
+
+
+@register_resources(_adjoint_pow_resource)
+def adjoint_pow_decomp(*_, base, **__):
+    """Decompose the adjoint of the power of a gate that has its own adjoint."""
+    qml.pow(base.base.adjoint(), z=base.z)
+
+
+def _pow_resource(base_class, base_params, z):
+    """Resources of the power of a gate."""
+    if not isinstance(z, int) or z < 0:
+        raise NotImplementedError("Non-integer or negative powers are not supported yet.")
+    return {resource_rep(base_class, **base_params): z}
```

Comment:
I guess this is what the dev comment regarding "full support of Pow" was about, but to make sure: There is a plan to improve upon this generic unrolling of pow ops? :)

Reply:
Yes, there is a general epic for add missing support to many things, `pow` being one of them. It is tracked.